tests/unit_tests/test_mesh_to_vtk.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from os.path import exists
+import pytest
+import vtk
+from vtk.util import numpy_support as nps
```

Comment:
You can use [`pytest.importorskip`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip) so that  these tests will get skipped if vtk is not available